File: Source/Aladdin.CAPI.ANSI/ASN1/ANSI/X962/ECPoints.cs
using System; 
using System.Runtime.Serialization;

namespace Aladdin.ASN1.ANSI.X962
{
    // ECPoints ::= SEQUENCE OF ECPoint

	[Serializable]
    public class ECPoints : Sequence<OctetString>
    {
		// конструктор при сериализации
        protected ECPoints(SerializationInfo info, StreamingContext context) : base(info, context) {}

	    // конструктор при раскодировании
	    public ECPoints(IEncodable encodable) : base(encodable) {} 

	    // конструктор при закодировании
	    public ECPoints(params OctetString[] values) : base(values) {} 
    }
}

File: Source/Aladdin.CAPI.ANSI.Java/src/aladdin/asn1/ansi/x962/ECPoints.java
package aladdin.asn1.ansi.x962;
import aladdin.asn1.*; 
import java.io.*;

// ECPoints ::= SEQUENCE OF ECPoint

public final class ECPoints extends Sequence<OctetString>
{
	// конструктор при раскодировании
	public ECPoints(IEncodable encodable) throws IOException 
	{
		super(OctetString.class, encodable); 
	} 
	// конструктор при закодировании
	public ECPoints(OctetString... values) 
	{
		super(OctetString.class, values); 
	} 
}
